src/normi/_estimators.py

```diff
@@ -94,15 +94,15 @@ class NormalizedMI(BaseEstimator):
     def __init__(
         self,
         *,
-        n_dims: PositiveInt = 1,
+        n_dims: Union[List[PositiveInt], PositiveInt] = 1,
```

Comment:
Instead of using `List[PositiveInt]` i guess it would be helpful to accept numpy arrays as well. This could be achieved by introducing a `ArrayLikeInt` and `ArrayLikePositiveInt` similar to https://github.com/moldyn/NorMI/blob/main/src/normi/_typing.py#L87-L88

Reply:
Numpy arrays are now accepted.